Resources/Prototypes/Language/languages.yml

```diff
@@ -31,6 +31,14 @@
   - nah
   - wah
 
+# The common galactic Sign Language!
+- type: language
+  id: SignLanguage
+  signLanguage: true
```

Comment:
So, two languages that are both Sign Languages cannot understand each other? That has some **very** cool implications.

Reply:
you can make multiple sign languages